frame/beefy/src/lib.rs

```diff
@@ -279,6 +299,30 @@ pub mod pallet {
 			Self::validate_unsigned(source, call)
 		}
 	}
+
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_initialize(_n: BlockNumberFor<T>) -> Weight {
+			// weight used by `on_finalize()` hook
+			frame_support::weights::constants::RocksDbWeight::get().reads(1)
```

Comment:
IIUC this isn't technically correct, because at BEEFY genesis block we'll be doing `3` db reads (`Authorities` and `ValidatorSetId` in addition to `GenesisBlock`). So probably it should be `frame_support::weights::constants::RocksDbWeight::get().reads(if GenesisBlock::<T>::get() == Some(n) { 3 } else { 1 })`. Don't think it is critical here, though :)